playlists/excel.yaml

```diff
@@ -487,6 +487,15 @@
   group: Preview APIs
   api_set:
     ExcelApi: 1.7
+- id: excel-events-table-changed
+  name: Events - Table changed
+  fileName: events-table-changed.yaml
+  description: Add event handlers for table onDataChanged and onSelectionChanged events
```

Comment:
The format (wording) of the description that you've used for the 3 event-related snippets differ. Suggest that we use consistent wording in snippet descriptions, as seeing the inconsistent wording in the menu of samples looks odd.

This snippet (**Events - Table changed**):  
- Add event handlers for table onDataChanged and onSelectionChanged events

**Events - Worksheet changed** snippet:
- Add event handlers on the data changed and worksheet added events

**Events - worksheet activated** snippet:
- Add an event handler on the worksheet activated and deactivated events